subsys/bluetooth/Kconfig.discovery

```diff
@@ -22,6 +22,25 @@ config BT_GATT_DM_DATA_PRINT
 	help
 	  Enable functions for printing discovery related data
 
+config BT_GATT_DM_WQ_STACK_SIZE
+	# Hidden option for workqueue stack size. Should be derived from system
+	# requirements.
+	int
+	default 1300 if BT_GATT_CACHING
+	default 1140 if BT_TINYCRYPT_ECC
+	default 1024
+
+config BT_GATT_DM_WQ_PRIO
+	int "GATT Discovery Manager workqueue priority. Should be pre-emptible."
+	default 10
+	range 0 NUM_PREEMPT_PRIORITIES
+
+config BT_GATT_DM_WQ_INIT_PRIO
+	int "GATT Discovery Manager workqueue init priority"
+	default 50
+	help
+	  Init priority level to setup the GATT Discovery Manager workqueue.
+
```

Comment:
These values are based on the "BT LONG WQ". Should we change them to something else, if so what?

Reply:
1. I think it does not matter if it's preemptible.
2. We should use some sort of default. Maybe copy it from the main thread or the system work queue.